packages/components/src/index.js

```diff
@@ -157,6 +158,10 @@ export {
 	Notification,
 	ObjectViewer,
 	Progress,
+	RichTooltip,
+	RichTooltipHeader,
+	RichTooltipBody,
+	RichTooltipFooter,
```

Comment:
please only export RichTooltip

Reply:
i need to export RichTooltipHeader, RichTooltipBody, RichTooltipFooter to allow cmf to connect them.